### DISCUSSION TITLE
Add "install requirements" in documentation

### DISCUSSION THREAD

**Discussion by DevBenLefevre** [cedricp/ddt4all#1118] [Ideas]
Hello !

First : THANK YOU !

Then : I may have bad installed the projet, but I installed every dependencies in the dependencies section of the documentation one by one, following the links.

Since, I had 
`
Traceback (most recent call last):
  File "/Users/b.lefevre/dev/ddt4all/main.py", line 15, in <module>
    import PyQt5.QtWebEngineWidgets as webkitwidgets
ModuleNotFoundError: No module named 'PyQt5.QtWebEngineWidgets'
`
first running the project, I had to run `python3 -m pip install -r requirements.txt` on MacOS.

I think it can be a good idea to add it to the documentation, probably in dependencies section.

**Comment by Furtif** [comment]
read https://github.com/cedricp/ddt4all/issues/320#issuecomment-680182687

**Comment by Furtif** [comment]
or https://github.com/cedricp/ddt4all/issues/1112
